tests/performance/test_wmg_apis.py

```diff
@@ -11,6 +12,8 @@
 
 # Note that these tests share fixtures and general test paths with the wmg api functional tests
 
+logger = logging.getLogger(__name__)
+
 
 @unittest.skipIf(os.getenv("DEPLOYMENT_STAGE") != "prod", "this test should only run in prod")
```

Comment:
Is this only running in prod due to potential timing differences caused by differences between the staging and prod infrastructure?

The problem with this is that we don't catch performance regressions until it's too late, i.e. prod deployment day. If the staging infra != prod infra, perhaps we should make them identical and also run this test in staging?

Reply:
Yea there is a difference in infrastructure and (prior to the env mirror work) the cube in dev/staging was much smaller so the perf would be way better in those envs/not actually useful for determining much about how it would perform in prod